.github/workflows/release_notes_comments_copy_from_pr_to_linked_issues.yml

```diff
@@ -0,0 +1,95 @@
+name: Release Notes Comments Copy from PR to Linked Issues
```

Comment:
some of these YML files have licence headers and some don't, what is the consensus around it? SHould it be present everywhere in OSS repositories?

Reply:
All should have it. Thanks